log-collector-script/linux/eks-log-collector.sh

```diff
@@ -544,6 +544,7 @@ get_system_services() {
 
   timeout 75 top -b -n 1 > "${COLLECT_DIR}"/system/top.txt 2>&1
   timeout 75 ps fauxwww > "${COLLECT_DIR}"/system/ps.txt 2>&1
+  timeout 75 ps -eTF > "${COLLECT_DIR}"/system/ps-threads.txt 2>&1
```

Comment:
Can you add `--headers` to the `ps` commands?

Reply:
They're printed once by default, do you want them printed after every page of output?

Reply:
Yeah, I find `ps` output hard to look at, lots of columns. 😂

Reply:
Sweet, thanks!